eng/pipelines/docindex.yml

```
@@ -114,6 +120,22 @@ jobs:
             TargetRepoOwner: $(DocRepoOwner)
             WorkingDirectory: $(DocRepoLocation)
 
+        - task: AzureCLI@2
+          displayName: Queue Docs CI build
```

In keeping with the style of the other steps... The pipeline queuing below should probably also include `for Daily docs`

```suggestion
          displayName: Queue Docs CI build for main branch
```